jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageLayers.java

```diff
@@ -19,34 +19,36 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
 import javax.annotation.Nullable;
 
 /** Holds the layers for an image. Makes sure that each layer is only added once. */
```

Comment:
We should remove "Makes sure that each layer is only added once" or update the Javadoc.